### DISCUSSION TITLE
onedrive国际版账户缓存文件缺少问题缺少

### DISCUSSION THREAD

**Discussion by DAIII8888** [libsgh/PanIndex#213] [Q&A]
**onedrive国际版缓存文件总是缓存不完整，总有些文件夹里文件显示网盘空空如也，但文件确实在，再来一次缓存这个文件夹缓存到了又到了下个文件夹里的文件没缓存到，阿里云盘没这情况，缓存一次就全部缓存到了，od国际版目前缓存了差不多十多次，每次都等个5-10分钟，缓存文件数量也每次都在变，缓存时间也很短，没一次缓存结果超过三分钟，每次有等一段时间才再次缓存**

**Comment by libsgh** [comment]
试试新版，账号要一个一个更新，因为更新是异步的，sqlite不支持并发写库
